File: solutions/058.length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int len = 0;
        char[] c = s.toCharArray();
        boolean start = false;
        for(int i = c.length - 1; i > -1; i--){
            if(c[i] != ' '){
                start = true;
                len++;
            }
            if(c[i] == ' ' && start){
                break;
            }
        }
        return len;
    }
}

File: solutions/058.length-of-last-word/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let start = -1;
    let len = 0;
    for(let i = s.length -1; i >= 0;i--){
        if(s[i] === ' '){
            if(start === -1){
                continue;
            }
            break;
        }else{
            if(start === -1) start = i;
            len++;
        }
    }
    return len;    
};